include/aspect/material_model/utilities.h

```diff
@@ -317,6 +320,32 @@ namespace aspect
                             const std::vector<double> &parameter_values,
                             const CompositionalAveragingOperation &average_type);
 
+
+
+      /**
+      * This function computes averages of multicomponent thermodynamic properties
+      * that are stored in a vector of EquationOfStateOutputs.
+      * Each EquationOfStateOutput contains the thermodynamic properties for
+      * all materials at a given evaluation point.
```

Comment:
What is "Each EquationOfStateOutput"? Below we have only one "EquationOfStateOutputs" object. If you mean that one, you could write 

```suggestion
      * @p eos_outputs  contains the thermodynamic properties for
      * all materials at a given evaluation point.
```